source/slang/slang-profile-defs.h

```diff
@@ -129,6 +129,8 @@ PROFILE(DX_Compute_6_0,				cs_6_0,				Compute,	DX_6_0)
 PROFILE(DX_Compute_6_1,				cs_6_1,				Compute,	DX_6_1)
 PROFILE(DX_Compute_6_2,				cs_6_2,				Compute,	DX_6_2)
 PROFILE(DX_Compute_6_3,				cs_6_3,				Compute,	DX_6_3)
+PROFILE(DX_Compute_6_4,				cs_6_4,				Compute,	DX_6_4)
+PROFILE(DX_Compute_6_5,				cs_6_5,				Compute,	DX_6_5)
```

Comment:
We eventually need to make a decision about whether we continue to define stage-specific `-profile` names for each new shader model (creating an O(N*M) problem) or if we eventually keep things separate and only support using separate `-stage compute -profile sm_6_5`.

The only reason to support these as their own profiles is for compatibility with users who work with dxc, so I guess we are stuck following that precedent for now.